asttokens/mark_tokens.py

```diff
@@ -195,9 +219,10 @@ def handle_attr(self, node, first_token, last_token):
   visit_delattr = handle_attr
 
   def handle_def(self, node, first_token, last_token):
+    # type: (AstNode, util.Token, util.Token) -> Tuple[util.Token, util.Token]
     # With astroid, nodes that start with a doc-string can have an empty body, in which case we
     # need to adjust the last token to include the doc string.
-    if not node.body and getattr(node, 'doc', None):
+    if not getattr(node, 'body', None) and getattr(node, 'doc', None):
```

Comment:
Why this change?

Reply:
This was mostly because it predated the work with narrowing the types for the various handle/visit functions down to the ones they were actually using. I tried doing that here, and the list of types is pretty large (I got to about at least 6 triggered by the tests), and they're annoyingly lacking in common parent classes to use instead. I've reset this back to it's original code and added a `type: ignore` comment.